app/frontends/workflow-ui.ts

```diff
@@ -138,11 +139,25 @@ function jobRenderingFunctions(logger: Logger, requestQuery: Record<string, any>
     },
     jobCreatedAt(): number { return this.createdAt.getTime(); },
     jobUpdatedAt(): number { return this.updatedAt.getTime(); },
+    jobRequest(): string {
+      try {
+        return decodeURIComponent(this.request);
+      } catch (e) {
+        logger.error(`Could not decode URL from job.request: ${this.request}`);
+        logger.error(e);
+        return this.request;
+      }
+    },
     jobUrl(): string {
       try {
         const url = new URL(this.request);
-        const path = url.pathname + url.search;
-        return truncateString(path, 100);
+        let { pathname } = url;
+        if (pathname.indexOf('ogc-api-coverages') > -1) {
+          pathname = pathname.replace(`${ogcVersion}/collections`, '...');
+          pathname = pathname.replace('coverage/rangeset', '...');
+        }
+        const path = pathname + decodeURIComponent(url.search);
+        return truncateString(path, 315);
```

Comment:
I can't tell if this is the cause, but the copy functionality is broken for large requests. We should definitely truncate on display, but not on the copy to clipboard.

Note that there are issues with really large query strings where only a POST will work, but ideally they would get an error back about the URL being too long for a GET than an error about an id with '...' not found in CMR.

Reply:
Great find. Made an update here: https://github.com/nasa/harmony/pull/454/commits/d2cffd6f151c63c3d2824f8c547723b5d897b7d0

Reply:
I'm trying to tweak the wording so that people who see a warning about the truncation don't think that there is something wrong with their job.

Reply:
If you can think of a better way to word the warning that let me know.

Reply:
Tested out successfully. I don't have a better recommendation, I usually go James for wordsmithing.